### PR TITLE
Login page for users to store their records

### PR DESCRIPTION
**Pull request by Shimrangupta22**
I have created a login or sign up page so that user can see their previous records.

### CODE REVIEW

File: game.html [hunk 1]
```diff
@@ -84,9 +85,18 @@ <h4>Javascript Game</h4>
             />
           </a>
         </div>
+        
         <button id="instructions-button" title="How to play?">
           Instructions
         </button>
+
```
Comment (Shimrangupta22) [inline game.html:92]: added a login button

File: game.html [hunk 2]
```diff
@@ -155,5 +165,38 @@ <h2>
         </div>
       </div>
     </div>
+
```
Comment (Shimrangupta22) [inline game.html:168]: Form code

File: style.css [hunk 3]
```diff
@@ -162,11 +162,28 @@ body {
 }
 
 /* CSS for the Instructions Button */
+.btn-login{
```
Comment (Shimrangupta22) [inline style.css:165]: CSS for login button added

**Comment by ayushi-ras** [comment]
can you add the screenshot how it looks?

**Comment by Shimrangupta22** [comment]
Yes, I can.

**Comment by Shimrangupta22** [comment]
Screenshot (17)

**Comment by Shimrangupta22** [comment]
@ayushi-ras mam give it a look

**Comment by ayushi-ras** [comment]
> @ayushi-ras mam give it a look

can you make it  full page like once the user login then only it shows the game interface?

**Comment by ayushi-ras** [comment]
Also there is a conflict:-
Conflicting files
game.html
style.css

**Comment by Shimrangupta22** [comment]
> > @ayushi-ras mam give it a look
> 
> can you make it full page like once the user login then only it shows the game interface?

@ayushi-ras yes mam , i can

**Comment by ayushi-ras** [comment]
Kindly solve the branch conflicts too . And add the screenshot of preview in the pr . @Shimrangupta22

**Pull request closed without merging**